### PR TITLE
Added posv

### PR DESCRIPTION
**Pull request by tfalders**
Addresses SWDEV-291330.

### CODE REVIEW

File: clients/include/testing_posv.hpp [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+    rocblas_stride stB = argus.get<rocblas_stride>("strideB", ldb * nrhs);
+
+    rocblas_fill uplo = char2rocblas_fill(uploC);
+    rocblas_int bc = argus.batch_count;
+    rocblas_int hot_calls = argus.iters;
+
+    rocblas_stride stBRes = (argus.unit_check || argus.norm_check) ? stB : 0;
+
+    // check non-supported values
+    // N/A
```
Comment (jzuniga-amd) [inline clients/include/testing_posv.hpp:320]: You need to verify that uplo != rocblas_fill_full as it is not supported...
This case is tested in badArgs( ) for the gtests, but a user can call rocsolver-bench with uplo=rocblas_fill_full and we must detect it and print corresponding message. See POTF2/POTRF for example. 
(Also, I forgot to add this comment to POTRS. You will need to change it as well)

**Review by jzuniga-amd — CHANGES_REQUESTED** [review]
Just one little change that I forgot to add here and in POTRS

**Review by jzuniga-amd — APPROVED** [review]
LGTM

**Review by cgmb — APPROVED** [review]
LGTM

**Comment by tfalders** [comment]
Some CI tests aren't passing because of the trsm bug, but others are so I'm going to go ahead and merge.

**Pull request merged**